program/Portuguese (Brazil)/exe.txt

```diff
@@ -2,4 +2,4 @@ IDS_OFFER_REJECTED      "A oferta de um programa potencialmente indesejado foi
 IDS_SOME_OFFERS_REJECTED 
                         "%d potencialmente foram rejeitadas as ofertas do programa indesejado!"
 IDS_OFFER_REJECTED_MORE "Clique aqui para mais detalhes..."
-IDS_BROWSER_WARNING     "Aviso de %s:\nAo aceitar isso, você pode instalar um programa que você não tinha a intenção"
+IDS_BROWSER_WARNING     "Aviso de %s:\Ao selecionar aqui, você pode instalar um programa que você não deseja"
```

Comment:
The '\n' combination means a newline. I'll fix it for you, but please don't remove it in the future. Thanks.